python/GafferArnold/__init__.py

```diff
@@ -42,18 +42,16 @@
 
 try :
 
-	# Make sure we import IECoreArnold and _GafferArnold
-	# _without_ RTLD_GLOBAL. This prevents clashes between the
-	# LLVM symbols in libai.so and the Mesa OpenGL driver.
-	# Ideally we wouldn't use RTLD_GLOBAL anywhere - see
+	# Make sure we _GafferArnold _without_ RTLD_GLOBAL. This prevents clashes
```

Comment:
> I'm assuming the reason why we no longer do this to IECoreArnold here is because as part of moving it from Cortex, we no longer compile it with RTLD_GLOBAL?

Ah, no, this was an error actually. It made sense to remove it in the earlier commit because `_IECoreArnold` no longer existed (it was just additional bindings in `_GafferArnold`). But not it is a standalone module, it needs to come back. I'll sort it out. I also need to get to the bottom of these Mac failures, and I think I can improve things by taking a more principled approach to the problem of the Arnold log and profile still being a global resource.

As an aside, RTLD_GLOBAL is a runtime thing, not a compile time thing. It gets turned on in `IECore/__init__.py` to work around historical problems, and I'm keen to remove it completely.

Reply:
Out of curiosity, what is holding us back from removing it? At IE we flipped it off back in March and haven’t had any issues.

Reply:
That's documented here : https://github.com/GafferHQ/gaffer/pull/4175. Problem with Appleseed, which is presumably why it hasn't affected IE?

Reply:
I fixed this in place here : https://github.com/GafferHQ/gaffer/pull/4461/commits/33d00d5b4f9fe04de89e67341b406d90352bbdec#diff-0de7066c8689f99f9be21d618e552ba306211985159e7b5f4b2d5f8277ea5a78R37-R50.